geode/src/test/resources/log4j2.xml

```diff
@@ -5,9 +5,9 @@
         <Property name="geode-default">true</Property>
     </Properties>
     <Appenders>
-        <Console name="STDOUT" target="SYSTEM_OUT">
+        <File name="File" fileName="target/geode-log4j.log">
```

Comment:
Couldn't we use a bridge jar to send log4j logging to slf4j/logback as well? Could be a separate pr though.

Reply:
Yes, that sounds easy, but this far I failed to find the correct mixture of logging dependencies as Geode uses Log4j**2** which has its own bridge to slf4j.

Reply:
The Geode `LogService` interacts directly with Log4j2 classes, not just the API.